rts/Rml/RmlInputReceiver.h

```diff
@@ -0,0 +1,21 @@
+/* This file is part of the Spring engine (GPL v2 or later), see LICENSE.html */
+
+#ifndef RMLINPUT_H
+#define RMLINPUT_H
+
+#include "Game/UI/InputReceiver.h"
+#include "Rml/Backends/RmlUi_Backend.h"
+
+class CRmlInputReceiver : public CInputReceiver
+{
+public:
+	CRmlInputReceiver() : CInputReceiver(FRONT), rml_active(false){};
+	~CRmlInputReceiver() = default;
+
+	bool IsAbove(int x, int y) { return rml_active; };
```

Comment:
shouldn't this check if any actual gui panel is underneath?

Reply:
Thinks like a dropdown combobox could drop below a gui panel?

Reply:
Sure, by "gui panel" I mean anything drawn by rml at the coordinates being asked about.

Reply:
`IsAbove` is used for determining which cursor icon should be displayed. I track if the mouse is over an rml element in `ProcessMouseEvent` and just return if rml is capturing the mouse since the xy is the mouse location.

Reply:
Good, put that in a comment at the function.

Reply:
I added the comment, and integrated the mouse event handling with the main mouse handler